src/main/java/org/jenkinsci/plugins/configfiles/folder/FolderConfigFileAction.java

```diff
@@ -21,26 +21,56 @@
 import org.jenkinsci.plugins.configfiles.Messages;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
-import org.kohsuke.stapler.*;
+import org.kohsuke.stapler.HttpRedirect;
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.QueryParameter;
+import org.kohsuke.stapler.StaplerProxy;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
 
-import javax.servlet.ServletException;
-import java.io.IOException;
-import java.util.*;
+import hudson.Extension;
+import hudson.Util;
+import hudson.model.Action;
+import hudson.model.Item;
+import hudson.model.Job;
+import hudson.security.ACL;
+import hudson.security.Permission;
+import hudson.util.FormValidation;
+
+import com.cloudbees.hudson.plugins.folder.AbstractFolder;
+
+import jenkins.model.TransientActionFactory;
+import net.sf.json.JSONObject;
```

Comment:
Probably a lot of this could have been not staged, and possibly filed as a separate PR. It's hard to see what is really belonging to that PR and what is just like `org.kohsuke.stapler.*` expansion and so on.